src/components/Navigation/SearchBar/SearchBar.vue

```diff
@@ -96,7 +97,7 @@ export default defineComponent({
 
     const setFoodOrder = function(event: any): void {
```

Comment:
U can switch it to arrow function, and type event

Reply:
Same as below- There is later problem with typing. I have consulted this earlier with Kevin, and he advised to let it be for now